addons/launcher-addon/src/main/java/io/fabric8/launcher/addon/preparers/ChangeArquillianConfigurationPreparer.java

```diff
@@ -0,0 +1,95 @@
+package io.fabric8.launcher.addon.preparers;
+
+import io.fabric8.launcher.booster.catalog.rhoar.RhoarBooster;
+import io.fabric8.launcher.core.api.CreateProjectileContext;
+import io.fabric8.launcher.core.api.ProjectileContext;
+import io.fabric8.launcher.core.spi.ProjectilePreparer;
+import java.io.File;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import javax.enterprise.context.ApplicationScoped;
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+import javax.xml.xpath.XPath;
+import javax.xml.xpath.XPathConstants;
+import javax.xml.xpath.XPathExpressionException;
+import javax.xml.xpath.XPathFactory;
+import org.w3c.dom.Document;
+import org.w3c.dom.NodeList;
+
+@ApplicationScoped
+public class ChangeArquillianConfigurationPreparer implements ProjectilePreparer {
+
+   private Logger LOG = Logger.getLogger(ChangeArquillianConfigurationPreparer.class.getName());
+
+   @Override
+   public void prepare(Path projectPath, RhoarBooster booster, ProjectileContext context) {
+      if (!(context instanceof CreateProjectileContext)) {
+         return;
+      }
+      CreateProjectileContext createProjectileContext = (CreateProjectileContext) context;
+      updateArquillianConfiguration(projectPath, createProjectileContext.getArtifactId());
+   }
+
+   void updateArquillianConfiguration(Path projectPath, String property) {
+      try {
+         Files.walk(projectPath)
+            .filter(path -> path.endsWith("arquillian.xml"))
```

Comment:
isn't easier to just do `projectPath.resolve("src/test/resources/arquillian.xml")`?